### ISSUE TITLE
white screen on notdoppler

### ISSUE THREAD

**Issue by flashliberty5467** [ruffle-rs/ruffle#4068]
games which work perfectly awesome on ruffle for some reason come up with a white screen on http://www.notdoppler.com/
i had to use SWF to HTML extension to work around the issue of white screens on not doppler they use ruffle in thier free extension
[`https://add0n.com/flashplayer.html](url)
[https://github.com/chandler-stimson/flashplayer](url)
**Describe the bug**
Provide a clear and concise description of what the bug is. If there are steps to reproduce, list them here.
If this is an issue with a specific SWF file, please provide a link to the SWF file, or you can attach the SWF by zipping it and dragging it onto the issue box.
If this is a visual issue, please attach a screenshot of the problem.
![drywhite](https://user-images.githubusercontent.com/64484245/115336871-9d6f0680-a165-11eb-9511-4bb3110de81b.png)

**Expected behavior**
for the game to load with normal ruffle extension but i am having to use the SWF to html extension to work around this issue
https://add0n.com/flashplayer.html

**Is the problem with the Ruffle desktop app, extension, or self-hosted version?** 
ruffle extension 2021-04-20
**What platform are you using?** ChromeOS
 - If you're using the desktop version, please list the OS you are using.
 - If you're using the web version, please list the OS, browser, and mobile device (if applicable).

 - ChromeOS Google Chrome

**Comment by danielhjacobs** [comment]
`Access to fetch at 'http://i.notdoppler.com/files/spacecraftescape.swf' from origin 'http://www.notdoppler.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.`

Possible duplicate of #2201?
